ONSdigital/eq-survey-runner
# Delete eb cloudwatch configuration

### What is the context of this PR?
When deploying to elastic beanstalk cloudwatch ingests one batch of events and then stops ingesting them. This fixes that.

### How to review 
1. Deploy aws infrastructure `./survey-runner.sh apply`
1. Deploy application `eb deploy`
1. Go to cloudwatch `https://eu-west-1.console.aws.amazon.com/cloudwatch/home`
1. Logs
1. Open error logs `/aws/elasticbeanstalk/$user-prime/var/log/httpd/error_log`
1. Check logs are being ingested after using your eq-survey-runner
